protocol/v2/ssv/validator/committee_queue.go

```diff
@@ -109,18 +109,24 @@ func (v *Committee) ConsumeQueue(
 			}
 		} else if runningInstance != nil && runningInstance.State.ProposalAcceptedForCurrentRound == nil {
 			// If no proposal was accepted for the current round, skip prepare & commit messages
-			// for the current height and round.
+			// for the current round.
 			filter = func(m *queue.SSVMessage) bool {
 				sm, ok := m.Body.(*specqbft.Message)
 				if !ok {
-					return true
+					return false
```

Comment:
I'm guessing this is wrong, we do want to prevent post-consensus at this stage like I intended, but this condition also applies to Event messages, while we know `ExecuteDuty` will not happen here as we move it to run through `StartDuty` we can still get a Timeout event message.